File: WebApi&angular/Dal/models/Table.cs
using System;
using System.Collections.Generic;

namespace Dal.models;

public partial class Table
{
    public int TableId { get; set; }

    public int TableNumber { get; set; }

    public int Seats { get; set; }

    public bool? IsOccupied { get; set; }
}

File: project/src/app/models/table.ts
export class Table{
     tableId: number;
  tableNumber: number;
  seats: number;
  isOccupied: boolean;

  constructor(tableId: number, tableNumber: number, seats: number, isOccupied: boolean){
    this.tableId = tableId;
    this.tableNumber = tableNumber;
    this.seats = seats;
    this.isOccupied = isOccupied;
  }
}